web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -31,6 +31,11 @@ const Form = (props: EntityFormProps): JSX.Element | null => {
       if (!aboutMe?.features.includes(feature.extraData?.iden as string)) {
         continue;
       }
+
+      if (feature.extraData?.iden === ClientFeatures.faxes) {
+        continue; //RETAIl ONLY
+      }
+
```

Comment:
I don't understand what this condition is checking. Also the last letter in retail is not capitalized :stuck_out_tongue_closed_eyes: